src/main/java/uk/gov/hmcts/reform/unspec/handler/callback/notification/DefendantResponseCaseHandedOfflineRespondentNotificationHandler.java

```diff
@@ -64,7 +65,7 @@ private CallbackResponse notifyClaimantSolicitorForCaseHandedOffline(CallbackPar
     public Map<String, String> addProperties(CaseData caseData) {
         return Map.of(
             CLAIM_REFERENCE_NUMBER, caseData.getLegacyCaseReference(),
-            SOLICITOR_REFERENCE, "claimant solicitor"
+            SOLICITOR_REFERENCE, "defendant solicitor"
```

Comment:
is this the reference we capture duering claim creation journey?

Reply:
it's a placeholder value, it will be updated when we'll have solicitors names available